apps/dailyscrum/src/app/page.tsx

```diff
@@ -13,17 +17,43 @@ export default async function Home() {
     redirect("/sign-in");
   }
 
+  const { data: profile, error: getProfileError } =
+    await getCurrentUserProfile();
+
+  if (!profile || getProfileError) {
+    redirect("/sign-up/complete");
+  }
+
   const { data: orgs, error } = await listOrgsWhereCurrentUserIsMember();
 
-  if (!orgs || error || orgs.length === 0) {
+  if (!orgs || error) {
+    return null;
+  }
+
+  if (orgs.length > 0) {
+    const [org] = orgs;
+
+    if (!org) {
+      return null;
+    }
+
+    return redirect(`/orgs/${org.id}`);
+  }
+
+  const { data: org, error: createOrgError } =
+    await createOrgWhereCurrentUserIsMember({
+      name: "My organization",
+    });
+
+  if (!org && createOrgError) {
     return null;
   }
 
-  const [org] = orgs;
+  const { error: initializeOrgError } = await initializeOrg(org.id);
```

Comment:
► The first organization creation and initialization are done on the root page conditionally.